### PR TITLE
chore: dependencies major version update - part 5

### PR DESCRIPTION
**Pull request by shahin-hq**
## Summary

As a part of: https://app.clickup.com/t/86drv24p3

There are too many file changes due to snapshot updates, I'll mark real changes to make review process easier.

- **cross-fetch** `3.1.8 -> 4.0.0`
  - 3 -> 4 - https://github.com/lquixada/cross-fetch/releases/tag/v4.0.0
- **twin.macro** `2.8.2 -> 3.4.0`
  - 2 -> 3 - https://github.com/ben-rogerson/twin.macro/releases/tag/3.0.0
  
Twins custom "short css" has been deprecated in favour of Tailwind arbitrary properties:

```
tw`backgroundColor[red]` // short css (no dash next to the `[`)
tw`[backgroundColor:red]` // Updated to an arbitrary property
```

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/app/components/Circle/Circle.styles.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const getAvatarCss = (avatarId?: string) => {
 		return;
 	}
 
-	return tw`border-0 background[#bad6f0]`;
+	return tw`border-0 [background:#bad6f0]`;
```
Comment (shahin-hq) [inline src/app/components/Circle/Circle.styles.ts:28]: Short css fix

File: src/app/components/Divider/Divider.styles.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ const isDashed = (dashed?: boolean) => {
 		return;
 	}
 
-	return tw`border-dashed background[none]`;
+	return tw`border-dashed [background:none]`;
```
Comment (shahin-hq) [inline src/app/components/Divider/Divider.styles.ts:37]: Short css fix

File: src/app/components/Header/HeaderSearchBar/HeaderSearchBar.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import { Input } from "@/app/components/Input";
 import { clickOutsideHandler, useDebounce } from "@/app/hooks";
 
 const SearchBarInputWrapper = styled.div`
-	${tw`min-width[300px] sm:min-width[448px] dark:border dark:border-theme-secondary-800`}
+	${tw`[min-width:300px] sm:[min-width:448px] dark:border dark:border-theme-secondary-800`}
```
Comment (shahin-hq) [inline src/app/components/Header/HeaderSearchBar/HeaderSearchBar.tsx:13]: Short css fix

File: src/app/components/Header/HeaderSearchInput/HeaderSearchInput.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { Input } from "@/app/components/Input";
 import { useDebounce } from "@/app/hooks";
 
 const SearchBarInputWrapper = styled.div`
-	${tw`min-width[300px] sm:min-width[448px] dark:border dark:border-theme-secondary-800`}
+	${tw`[min-width:300px] sm:[min-width:448px] dark:border dark:border-theme-secondary-800`}
```
Comment (shahin-hq) [inline src/app/components/Header/HeaderSearchInput/HeaderSearchInput.tsx:12]: Short css fix

File: src/app/components/Input/Input.tsx [hunk 5]
```diff
@@ -81,7 +81,7 @@ export const InputWrapperStyled = styled.div<{
 		}
 
 		if (isCompact) {
-			return tw`height[34px] overflow-hidden`;
+			return tw`[height:34px] overflow-hidden`;
```
Comment (shahin-hq) [inline src/app/components/Input/Input.tsx:84]: Short css fix

File: src/app/components/Link/Link.tsx [hunk 6]
```diff
@@ -29,9 +29,9 @@ const AnchorStyled = styled.a<{ isDisabled?: boolean }>(({ isDisabled }) => {
 const Content = styled.span<{ isDisabled?: boolean }>(({ isDisabled }) => {
 	const styles = [
 		tw`break-all border-b border-transparent`,
-		tw`transition-property[color, border-color]`,
-		tw`transition-duration[200ms, 350ms]`,
-		tw`transition-delay[0s, 100ms]`,
+		tw`[transition-property:color,_border-color]`,
+		tw`[transition-duration:200ms,_350ms]`,
+		tw`[transition-delay:0s, _100ms]`,
 	];
```
Comment (shahin-hq) [inline src/app/components/Link/Link.tsx:35]: Short css fix

File: src/app/components/Spinner/Spinner.styles.ts [hunk 7]
```diff
@@ -2,7 +2,7 @@ import tw, { TwStyle, css } from "twin.macro";
 
 import { Color, Size, Theme } from "@/types";
 
-const baseStyle = tw`animate-spin rounded-full border border-width[5px] flex-shrink-0`;
+const baseStyle = tw`animate-spin rounded-full border [border-width:5px] flex-shrink-0`;
```
Comment (shahin-hq) [inline src/app/components/Spinner/Spinner.styles.ts:5]: Short css fix

File: vitest.setup.ts [hunk 8]
```diff
@@ -79,6 +79,8 @@ const originalLocalStorageGetItem = localStorage.getItem;
 let localstorageSpy;
 
 beforeAll(async () => {
```
Comment (shahin-hq) [inline vitest.setup.ts:81]: https://github.com/ArdentHQ/arkvault/pull/523/files#diff-0a7996d580fb7938335496ed892e3c0a70af5770fdce11c375de28b9ec7b43cdR92

Got the same error and applied the given fix.

**Review by ItsANameToo — APPROVED** [review]

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**